### PR TITLE
Move components for Our Team and Publications pages to lib-content

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
lib-content
app-root

## Linked Issue and/or Talk Post
Toward: https://github.com/zooniverse/front-end-monorepo/issues/6023

## Describe your changes
- Move code for Publications page and Our Team page components into lib-content from app-content-pages
- Fetch contentful data server side in app-root
- Import Publications screen and OurTeam screen into app-root and pass data to them

## How to Review

- Most code for these two pages is copy-pasted into lib-content. Notable differences are:
    - Renamed Category to Discipline to match its UI label
    - Renamed Team to Institution to match its UI label and reduce confusion between that component and the OurTeam page
    - I consolidated helper functions for `publicationsData` into one file
    - There are no tests in app-root, which means the helper functions for processing contentful data are not covered. I decided not to add a test suite in app-root for contentful data because it would inflate this PR.

# Checklist
- Note the new comment in app-root's Readme about an `.env` file.
- To review you should be able to:
    - [x] Run lib-content's storybook and view new components there
    - [x] Visit https://local.zooniverse.org:3000/about/team
    - [x] Visit https://local.zooniverse.org:3000/about/publications
    - [x] Navigated to the other /about pages, too

## General
- [x] Tests are passing locally and on Github
- [x] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)

### CODE REVIEW

File: packages/app-root/src/app/about/publications/page.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import mockResponse from './response.mock.json'
+
+export const metadata = {
+  title: 'Our Team',
+  description: 'The people who make the Zooniverse'
+}
+
+// This route is static, so the output of the request will be cached and revalidated as part of the route segment.
+async function createPublicationsResponse() {
+  if (client) {
```
Comment (mcbouslog) [inline packages/app-root/src/app/about/publications/page.js:20]: It's unlikely there'd be an issue, but we could wrap the if statement with a try-catch and return the mockResponse in the catch as well. Not blocking, for consideration.

File: packages/app-root/src/app/about/publications/page.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+export const revalidate = 3600 // revalidate the data at most every hour
+
+export default async function PublicationsPage() {
+  const publicationsData = await createPublicationsResponse()
+
+  // For rendering linked h2's
+  publicationsData?.forEach(
+    category =>
+      (category.slug = category.title.toLowerCase().replaceAll(' ', '-'))
```
Comment (mcbouslog) [inline packages/app-root/src/app/about/publications/page.js:39]: Not sure if some optional chaining here would have helped with the recent CI build errors, or if preferable to have the errors so we know to fix.

File: packages/app-root/src/app/about/team/response.mock.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "name": "Alumni",
+    "weight": 100,
+    "people": [
+      {
+        "bio": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        "jobTitle": "Project Assistant",
+        "name": "Mock Person One",
+        "avatarSrc": "https://placekitten.com/300/300?w=160&h=160"
```
Comment (mcbouslog) [inline packages/app-root/src/app/about/team/response.mock.json:10]: When I run `yarn build` then `yarn start` and check https://local.zooniverse.org:3000/about/team the avatar images are not displayed.

**Review by mcbouslog — APPROVED** [review]
Nothing blocking, looking good! 🚀 

To confirm my understanding - Category and Team components/filenames were renamed to Discipline and Institution, respectively, but the code we'll leave as is.

File: packages/app-root/src/app/about/publications/page.js (hunk 2 above)
Comment (goplayoutside3) [reply packages/app-root/src/app/about/publications/page.js:39]: Good point I'll add that!

File: packages/app-root/src/app/about/team/response.mock.json (hunk 3 above)
Comment (goplayoutside3) [reply packages/app-root/src/app/about/team/response.mock.json:10]: placekitten.com has been out of commission for awhile 😢 I'll choose a different url

File: packages/app-root/src/app/about/publications/page.js (hunk 1 above)
Comment (goplayoutside3) [reply packages/app-root/src/app/about/publications/page.js:20]: True!

**Pull request merged**